_data/harnesses/anomaly-detection/ksql.yml

```diff
@@ -0,0 +1,125 @@
+dev:
+  steps:
+    - title: Initialize the project
+      content:
+        - action: execute
+          file: tutorial-steps/dev/init.sh
+          render:
+            file: tutorials/anomaly-detection/ksql/markup/dev/init.adoc
+
+        - change_directory: anomaly-detection
+          action: execute
+          file: tutorial-steps/dev/make-dirs.sh
+          render:
+            file: tutorials/anomaly-detection/ksql/markup/dev/make-dirs.adoc
+
+    - title: Get Confluent Platform
+      content:
+        - action: make_file
+          file: docker-compose.yml
+          render:
+            file: tutorials/anomaly-detection/ksql/markup/dev/make-docker-compose.adoc
+
+        - action: execute_async
+          file: tutorial-steps/dev/docker-compose-up.sh
+          render:
+            file: tutorials/anomaly-detection/ksql/markup/dev/start-compose.adoc
+
+        - action: execute
+          file: tutorial-steps/dev/wait-for-containers.sh
+          render:
+            skip: true
+
+    - title: Write the program interactively using the CLI
```

Comment:
Not sure if there's a convention or not, but most of the meat in this tutorial is in this section.  Is there value in breaking this up?

Reply:
I did move forward with having this broken down into new sections. There is value but also some obstacles of keeping ksqldb cli configurations persistent through the newly created ksqldb cli sessions that are started with each new section. Talked about in [slack on this thread](https://confluent.slack.com/archives/GT12FQG9E/p1603738429008700). This is generally not done for ksqlDB tutorials, I would guess due to the hurdles of starting new cli sessions.